go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/pkg/errors v0.8.0
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/prometheus/client_golang v1.18.0
+	github.com/prometheus/client_model v0.5.0
```

Comment:
I guess a `go mod tidy` was missed in one of the latest metrics PR ?

Reply:
Here https://github.com/vechain/thor/pull/758/files